commitizen-tools/commitizen
# Sign tag

## Description
Add tag sign feature for bump command.


## Checklist


## Expected behavior

When added the `--signed-tag`, bump command will sign the tag with preconfigured GPG private key.

## Steps to Test This Pull Request

1. run `cz bump --signed-tag <tag>`
2. check with `git tag -v <tag>` that the stdout answer starts with `gpg: Signature made` string

A new method to check if the tag is signed was added to do same step of point 2 above.

## Additional context
